scripts/generate-docs.js

```diff
@@ -11,7 +11,7 @@ var docs = grep('^//@', 'shell.js');
 
 // Insert the docs for all the registered commands
 docs = docs.replace(/\/\/@commands\n/g, function () {
-  return require('../commands.json').map(function (commandName) {
+  return require('../commands').list.map(function (commandName) {
```

Comment:
If `module.exports` is used in `commands.js`, this will become `require('../commands').map(...)`.